subprojects/packagefiles/lmdb/meson.build

```diff
@@ -0,0 +1,29 @@
+project('lmdb', 'c', version: '0.9.29', license: 'OpenLDAP')
+
+cc = meson.get_compiler('c')
+
+src = [
+  'libraries/liblmdb/mdb.c',
+  'libraries/liblmdb/midl.c',
+]
+
+c_args = [
+  '-W',
+  '-Wall',
+  '-Wno-unused-parameter',
+  '-Wbad-function-cast',
+  '-Wuninitialized'
+]
+
+pthread_dep = cc.find_library('pthread', required: true)
+incdir = include_directories('.')
+
+lmdb = library(
+  'lmdb',
+  src,
+  c_args: c_args,
+  dependencies: pthread_dep,
+  include_directories: incdir,
```

Comment:
You don't ever need to add `'.'` as an include directories, since meson does that by default.

You also do not need to add any include directory at all, for the compiler to find a header in the same directory as the source file itself is located in.